---
date: 2016-02-18T10:58:08-04:00
title: "Establishing the precise evolutionary history of a gene improves predicting disease-causing missense mutations."
pubDate: 2016-02-18
description: "Research Article"
journal: "Genetics in Medicine"
authors: ["Adebali O.", "Reznik A.O.", "Ory D.S.", "Zhulin, I.B."]
members: [1]
equalCont: []
corresponding: [4]
doi: ""
pubmedId: ""
issue: 10
vol: 18
pages: 1029-1036
doi: 10.1038/gim.2015.208
# featured_image: "/images/Pope-Edouard-de-Beaumont-1844.jpg"
tags: ["research", "medicine", "evolution"]
category: "article"
draft: False
---

Predicting the phenotypic effects of mutations has become an important application in clinical genetic diagnostics. Computational tools evaluate the behavior of the variant over evolutionary time and assume that variations seen during the course of evolution are probably benign in humans. However, current tools do not take into account orthologous/paralogous relationships. Paralogs have dramatically different roles in Mendelian diseases. For example, whereas inactivating mutations in the NPC1 gene cause the neurodegenerative disorder Niemann-Pick C, inactivating mutations in its paralog NPC1L1 are not disease-causing and, moreover, are implicated in protection from coronary heart disease.

We identified major events in NPC1 evolution and revealed and compared orthologs and paralogs of the human NPC1 gene through phylogenetic and protein sequence analyses. We predicted whether an amino acid substitution affects protein function by reducing the organismâ€™s fitness.

Removing the paralogs and distant homologs improved the overall performance of categorizing disease-causing and benign amino acid substitutions.

The results show that a thorough evolutionary analysis followed by identification of orthologs improves the accuracy in predicting disease-causing missense mutations. We anticipate that this approach will be used as a reference in the interpretation of variants in other genetic diseases as well.